client/blocks/login/two-factor-authentication/verification-code-form.jsx

```diff
@@ -109,10 +121,15 @@ class VerificationCodeForm extends Component {
 						</FormLabel>
 
 						<FormTextInput
+							autoComplete="off"
+							autoFocus
 							value={ this.state.twoStepCode }
 							onChange={ this.onChangeField }
 							className={ classNames( { 'is-error': isError } ) }
-							name="twoStepCode" />
+							name="twoStepCode"
+							pattern="[0-9]*"
+							ref={ this.saveRef }
+							type="tel" />
```

Comment:
Minor, we should rather use `number` type here. This is a code, not a phone number :)